wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/CommandBase.java

```diff
@@ -105,5 +99,15 @@ public void initSendable(SendableBuilder builder) {
         });
     builder.addBooleanProperty(
         ".isParented", () -> CommandGroupBase.getGroupedCommands().contains(this), null);
+    builder.addStringProperty(
+        "interruptBehavior", () -> getInterruptionBehavior().toString(), null);
+    builder.addBooleanProperty("runsWhenDisabled", this::runsWhenDisabled, null);
+    builder.addStringArrayProperty(
+        "requirements",
+        () ->
+            getRequirements().stream()
+                .map(req -> req.getClass().getSimpleName())
+                .toArray(String[]::new),
```

Comment:
I'm a bit worried about the overhead of this since it's going to be called every periodic loop (ditto the other places which use a stream()/map() setup).

Reply:
Any suggestions?

Reply:
Does the C++ vector/for-loop method also have an overhead problem?
I'm wary of pre-computing the names because I'm not sure they're immutable.